Marlin/Marlin.h

```diff
@@ -35,13 +35,11 @@
 
 #include "MarlinConfig.h"
 
-#include "fastio.h"
-
 #include "enum.h"
+#include "types.h"
+#include "fastio.h"
```

Comment:
Cool. But note that "fastio.h" is already included at the top of `MarlinConfig.h`.